tests/framework/utils_cpuid.py

```diff
@@ -10,6 +10,8 @@
 from framework.utils import check_output
 from framework.utils_imdsv2 import imdsv2_get
 
+CPU_FEATURES_CMD = r"lscpu |grep -oP '^Flags:\s+\K.+'"
```

Comment:
Only used in one test, so I don't think it needs to be here.

Reply:
Idea is that it might be used in x86 version of this test. I did not implement it here though.

Reply:
In that case we can wait until that happens, but in general this test is simple enough that could work for both architectures.

Reply:
I added x86_64 test as well, so this global constant is now used in 2 places.